openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1117,6 +1118,53 @@ Cache-Control: no-store
 }
 ```
 
+# Callback Endpoint {#callback_endpoint}
+
+This endpoint is used to receive notification from the Wallet whether credential has been successfully received in the Wallet or not. Support for this endpoint is OPTIONAL.
+
+This endpoint can be used after the Credential Issuer has sent Credential Response or Batch Credential Response. It enables the Credential Issuer to take subsequent actions after issuance, depending on whether the credential has been accepted and successully stored by the Wallet, rejected by the Wallet, or errors and other unforeseen circumstances have occurred during the Wallet's processing.
+
+Communication with the Callback Endpoint MUST utilize TLS.
+
+## Callback from the Wallet
+
+The Wallet sends a callback to the Callback Endpoint by sending the following parameters in the entity-body of an HTTP POST request using the `application/json` media type.
+
+* `status`: REQUIRED. Status whether the credential issuance was successful or not. The value MUST be either `success` or `error`.
+* `error_description`: OPTIONAL. Human-readable ASCII [@!USASCII] text providing additional information, used to assist the Credential Issuer developer in understanding the error that occurred. Values for the `error_description`` parameter MUST NOT include characters outside the set %x20-21 / %x23-5B / %x5D-7E.
+
+Below is a non-normative example of a callback when credential issuance was successful:
+
+```
+POST /callback HTTP/1.1
+Host: server.example.com
+Content-Type: application/json
+{
+  "status": "success"
+}
+```
+
+Below is a non-normative example of a callback when credential issuance was unsuccessful:
+
+```
+POST /callback HTTP/1.1
+Host: server.example.com
+Content-Type: application/json
+{
+  "status": "error"
+  "error_description": "..."
+}
+```
+
+## Response from the Credential Issuer
+The Credential Issuer MUST respond with the HTTP response with a 200 (OK) status code using the `text/plain` media type:
+```
+HTTP/1.1 200 OK
+Content-Type: text/plain
+OK
```

Comment:
It Is required the presence of this string in the message body?

Reply:
May we assume that the http status code Is enough?

Reply:
```suggestion
```
good point.